### PR TITLE
Multiple binding markdown fixes

### PR DESCRIPTION
**Pull request by lsiepel**
- Fixes some SAT warnings about markdown. 
- Reformatted some tables. (no change to content, just whitespaces)
- Fixes some icons missing in the `docs` folder cuasing 404's.
- Fixes some duplicate headers in the docs, causing anchor tags to fail.
- Adjusted all lists to dashes
- Removed all trailing whitespaces
- Added ini language to some configuraiton related code fences

### CODE REVIEW

File: bundles/org.openhab.binding.warmup/README.md [hunk 1]
```diff
@@ -50,15 +48,15 @@ Once credentials are successfully added to the bridge, any rooms (devices) detec
...
+### `my-warmup` Bridge Configuration
 
 | config parameter | type    | description                                     | required | default |
 |------------------|---------|-------------------------------------------------|----------|---------|
 | username         | String  | Username for my.warmup.com                      | true     |         |
 | password         | String  | Password for my.warmup.com                      | true     |         |
 | refreshInterval  | Integer | Interval in seconds between automatic refreshes | true     | 300     |
 
-### Room
+### `Room` Thing Configuration
```
Comment (lolodomo) [inline bundles/org.openhab.binding.warmup/README.md:59]: Maybe rather room all in lowercase as it is the real thing type ?

File: bundles/org.openhab.io.homekit/README.md [hunk 2]
```diff
@@ -87,7 +86,7 @@ At the very least, you cannot use repeating (111-11-111) or sequential (123-45-6
 
 Other settings, such as using Fahrenheit temperatures and specifying the interface to advertise the HomeKit bridge are also illustrated in the following sample:
 
-```
+```perl
```
Comment (lolodomo) [inline bundles/org.openhab.io.homekit/README.md:89]: Why perl ?

Same question for the other places where you added it.

File: bundles/org.openhab.io.openhabcloud/README.md [hunk 3]
```diff
@@ -93,57 +93,57 @@ To send push notifications, the notification actions have to be used in rules.
 
 Three different actions are available:
 
-- `sendNotification(emailAddress, message)`: Send a notification to a _specific_ openHAB Cloud user.
-- `sendBroadcastNotification(message)`: Send a broadcast notification to _all_ devices of _all_ users.
-- `sendLogNotification(message)`: Send a log notification to the notifications list. Log notifications do not trigger a notification on the device.
+* `sendNotification(emailAddress, message)`: Send a notification to a _specific_ openHAB Cloud user.
```
Comment (lolodomo) [inline bundles/org.openhab.io.openhabcloud/README.md:96]: Why in the previous file (homekit) you replaced * by - and here you do the reverse. The rendering seems to be identical, no ?

File: bundles/org.openhab.binding.warmup/README.md (hunk 1 above)
Comment (lsiepel) [reply bundles/org.openhab.binding.warmup/README.md:59]: Yes, thanks!

File: bundles/org.openhab.io.homekit/README.md (hunk 2 above)
Comment (lsiepel) [reply bundles/org.openhab.io.homekit/README.md:89]: `.cfg` files are not listed in the post that is used as base (for reference: https://github.com/openhab/openhab-docs/issues/999)
As there does not seem to be a preferred one, i choose this as it seems a decent fit. For instance it uses the same comment `#`
But i'm, open for other suggestions.

**Comment by holgerfriedrich** [comment]
Overall, LGTM, thanks for caring!

I share the doubts with perl code formatting, but I have not checked the rendering. Maybe not a problem at all - though we typically use java formatting, right?

The question is if we should change meteoalerte (as my impression is that it should be replaced by metrofrance, correct me if I am wrong).

**Comment by lsiepel** [comment]
> Overall, LGTM, thanks for caring!
> 
> I share the doubts with perl code formatting, but I have not checked the rendering. Maybe not a problem at all - though we typically use java formatting, right?

I can revert those perl changes and maybe @wborn can add anything to it as he previously created the base in the linked post for these languages. As Java uses a different comment tag I would opt not to use that case fence language.

> The question is if we should change meteoalerte (as my impression is that it should be replaced by metrofrance, correct me if I am wrong).

I’ll revert changes to that one as we are about to finish the change from meteoalerte to meteofrance with @clinique

**Comment by lsiepel** [comment]
Reverted the perl blocks as these are doubts and reverted meteoalerte.
Please merge to prevent conflicts as it touches multiple bindings.

**Review by holgerfriedrich — APPROVED** [review]

**Comment by lsiepel** [comment]
Build fail is not related @lolodomo

**Comment by lsiepel** [comment]
@openhab/add-ons-maintainers I prefer not to go though conflict resolving so if you have some time, please merge.

File: bundles/org.openhab.io.openhabcloud/README.md (hunk 3 above)
Comment (jlaur) [reply bundles/org.openhab.io.openhabcloud/README.md:96]: It should be "- " rather than "* ", see for example #14371.

Comment (lsiepel) [reply bundles/org.openhab.io.openhabcloud/README.md:96]: I’ll adapt, as being repo wide consistent is a good thing. For the warnings, lifting or rendering it does not matter. 
I’ll push the changes tomorrow morning

File: bundles/org.openhab.io.openhabcloud/README.md [hunk 4]
```diff
@@ -70,7 +70,7 @@ Alternatively, you can configure the settings in the file `conf/services/openhab
 # - notification: Only push notifications are enabled, no remote access is allowed.
 # - remote: Push notifications and remote access are enabled.
 # Optional, default is 'remote'.
-#mode=
+mode=
```
Comment (jlaur) [inline bundles/org.openhab.io.openhabcloud/README.md:73]: Why this change? I would guess that it was commented on purpose (like `#baseURL=` above) so it could be left untouched and defaulted.

File: bundles/org.openhab.io.openhabcloud/README.md (hunk 3 above)
Comment (jlaur) [reply bundles/org.openhab.io.openhabcloud/README.md:96]: Yes, you are right that is doesn't have any rendering impact, I believe the previous documentation refactoring was for consistency.

File: bundles/org.openhab.io.openhabcloud/README.md (hunk 4 above)
Comment (lsiepel) [reply bundles/org.openhab.io.openhabcloud/README.md:73]: Thanks this was not meant to be included, I’ll revert this. I remember I changed it to see if the specified language properly did code highlighting with comments

File: bundles/org.openhab.voice.googletts/README.md [hunk 5]
```diff
@@ -77,7 +77,9 @@ In case you would like to setup these settings via a text file, you can edit the
 org.openhab.voice:defaultTTS=googletts
 org.openhab.voice:defaultVoice=googletts:XXX
 ```
+
```
Comment (jlaur) [inline bundles/org.openhab.voice.googletts/README.md:80]: I saw your reverted commit using \`\`\`perl above. I'm wondering if \`\`\`ini would be useful here?

Comment (lsiepel) [reply bundles/org.openhab.voice.googletts/README.md:80]: Looks very similar for these blocks. I prefer to first discuss this before I create a new (separate) PR. @Confectrician might be able to add something too.

Comment (jlaur) [reply bundles/org.openhab.voice.googletts/README.md:80]: It seems used already:

```shell
$ grep -A 2 -R "\`\`\`ini"
bundles/org.openhab.automation.jrubyscripting/README.md:   ```ini
bundles/org.openhab.automation.jrubyscripting/README.md-   org.openhab.automation.jrubyscripting:gems=openhab-scripting=~>5.0
bundles/org.openhab.automation.jrubyscripting/README.md-   org.openhab.automation.jrubyscripting:require=openhab/dsl
--
bundles/org.openhab.automation.jrubyscripting/README.md:```ini
bundles/org.openhab.automation.jrubyscripting/README.md-org.openhab.automation.jrubyscripting:gems=openhab-scripting=~>5.0
bundles/org.openhab.automation.jrubyscripting/README.md-org.openhab.automation.jrubyscripting:require=openhab/dsl
--
bundles/org.openhab.binding.lirc/README.md:```ini
bundles/org.openhab.binding.lirc/README.md-[Unit]
bundles/org.openhab.binding.lirc/README.md-Description=Linux Infrared Remote Control
--
bundles/org.openhab.persistence.dynamodb/README.md:```ini
bundles/org.openhab.persistence.dynamodb/README.md-profilesConfigFile=/etc/openhab2/aws_creds
bundles/org.openhab.persistence.dynamodb/README.md-profile=fooprofile
--
bundles/org.openhab.persistence.dynamodb/README.md:````ini
bundles/org.openhab.persistence.dynamodb/README.md-[fooprofile]
bundles/org.openhab.persistence.dynamodb/README.md-aws_access_key_id=testAccessKey
```

and rendered nicely also:

- https://www.openhab.org/addons/bindings/lirc/#supported-things
- https://www.openhab.org/addons/automation/jrubyscripting/#using-files

Comment (lsiepel) [reply bundles/org.openhab.voice.googletts/README.md:80]: Cool! Will adapt

File: bundles/org.openhab.io.openhabcloud/README.md (hunk 3 above)
Comment (lsiepel) [reply bundles/org.openhab.io.openhabcloud/README.md:96]: Changed

File: bundles/org.openhab.voice.googletts/README.md (hunk 5 above)
Comment (lsiepel) [reply bundles/org.openhab.voice.googletts/README.md:80]: Done

**Review by wborn — APPROVED** [review]
Nice!

**Comment by lsiepel** [comment]
Again build fail is unrelated.

File: bundles/org.openhab.io.openhabcloud/README.md [hunk 6]
```diff
@@ -107,10 +107,10 @@ Icon and tag can potentially be used by cloud instance clients (such as the open
...
 The parameters for these actions have the following meaning:
 
-- `emailAddress`: String containing the email address the target user is registered with in the cloud instance.
-- `message`: String containing the notification message text.
-- `icon`: String containing the icon name (as described in [Items: Icons]({{base}}/configuration/items.html#icons)).
-- `tag`: String containing the tag for the notification.
+* `emailAddress`: String containing the email address the target user is registered with in the cloud instance.
+* `message`: String containing the notification message text.
+* `icon`: String containing the icon name (as described in [Items: Icons]({{base}}/configuration/items.html#icons)).
+* `tag`: String containing the tag for the notification.
```
Comment (jlaur) [inline bundles/org.openhab.io.openhabcloud/README.md:113]:
```suggestion
- `emailAddress`: String containing the email address the target user is registered with in the cloud instance.
- `message`: String containing the notification message text.
- `icon`: String containing the icon name (as described in [Items: Icons]({{base}}/configuration/items.html#icons)).
- `tag`: String containing the tag for the notification.
```

File: bundles/org.openhab.binding.tradfri/README.md [hunk 7]
```diff
@@ -79,6 +79,7 @@ The control outlet supports the `power` channel.
 A blind or curtain supports, beside `battery_level` and `battery_low` channels,  a `positon` channel.
 
 An air purifier supports:
+
```
Comment (jlaur) [inline bundles/org.openhab.binding.tradfri/README.md:82]: Should the list below also be changed to "- "?

**Review by morph166955 — APPROVED** [review]
LGTM AndroidTV and Neeo

File: bundles/org.openhab.binding.tradfri/README.md (hunk 7 above)
Comment (lsiepel) [reply bundles/org.openhab.binding.tradfri/README.md:82]: You made me go through many files ;-) i adapted as many as i could find.

**Comment by lsiepel** [comment]
Last commit is about removing trailing spaces using the regex "\s+$" 1189 occurences are fixed. I checked 15-20% and could not finde any reason why these whitespaces should be left in.

**Review by andrewfg — APPROVED** [review]
LGTM growatt
LGTM hue
LGTM neohub

**Review by florian-h05 — APPROVED** [review]
LGTM evcc

**Review by stefan-hoehn — APPROVED** [review]
Nanoleaf and Govee look good to me

**Review by janvyb — APPROVED** [review]

**Review by mlobstein — APPROVED** [review]
LGTM

**Review by jlaur — APPROVED** [review]
LGTM. Many thanks for the major cleanup.

**Review by druciak — COMMENTED** [review]
Satel looks good

**Review by psmedley — APPROVED** [review]
